.gitignore

```diff
@@ -1,2 +1,5 @@
 /composer.lock
 /vendor/
+/.idea
```

Comment:
You can add this to a local global gitignore file so that we don't have to put these editor specific kind of things into a projects gitignore file.

Reply:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore